--- memc3_infrastructure.orig.vhd	2010-05-29 12:15:17.000000000 +0200
+++ memc3_infrastructure.vhd	2010-05-29 12:17:39.000000000 +0200
@@ -118,7 +118,6 @@
   signal   clkfbout_clkfbin    : std_logic;
   signal   locked              : std_logic;
   signal   rst_tmp             : std_logic;
-  signal   sys_clk_ibufg       : std_logic;
   signal   sys_rst             : std_logic;
   signal   rst0_sync_r         : std_logic_vector(RST_SYNC_NUM-1 downto 0);
 
@@ -135,25 +134,6 @@
 
 
 
-diff_input_clk : if(C_INPUT_CLK_TYPE = "DIFFERENTIAL") generate   
-    u_ibufg_sys_clk : IBUFGDS
-      port map (
-        I  => sys_clk_p,
-        IB => sys_clk_n,
-        O  => sys_clk_ibufg
-        );
-end generate;   
-
-
-se_input_clk : if(C_INPUT_CLK_TYPE = "SINGLE_ENDED") generate   
-    u_ibufg_sys_clk : IBUFG
-      port map (
-        I  => sys_clk,
-        O  => sys_clk_ibufg
-        );
-end generate;   
-
-
   --***************************************************************************
   -- Global clock generation and distribution
   --***************************************************************************
@@ -192,7 +172,7 @@
           (
            CLKFBIN          => clkfbout_clkfbin,
            CLKINSEL         => '1',
-           CLKIN1           => sys_clk_ibufg,
+           CLKIN1           => sys_clk,
            CLKIN2           => '0',
            DADDR            => (others => '0'),
            DCLK             => '0',
